extensions/roc-plugin-dredd/src/actions/dredd.js

```diff
@@ -0,0 +1,33 @@
+import { initLog } from 'roc';
+import Dredd from 'dredd';
+
+const log = initLog();
+
+export default ({ context: { config: { settings } } }) => () => {
+    const port = process.env.PORT || settings.runtime.port;
+
+    const dredd = new Dredd({
+        server: `http://localhost:${port}`,
```

Comment:
I suggest using the `port` and `path` that the action gets as arguments.

https://github.com/rocjs/roc-package-web-app/blob/master/extensions/roc-package-web-app-dev/docs/Hooks.md#server-started

I would also in this case put all of the three nested callbacks together, something like this.

```js
export default () => (port, path) => () => {
  const dredd = new Dredd({
```